content/main.js

```diff
@@ -128,12 +130,19 @@ var tblatex = {
       var latex_bin = init_file(prefs.getCharPref("latex_path"));
       if (!latex_bin.exists()) {
         log += "Wrong path for latex bin. Please set the right path in the options dialog first.\n";
-        return [2, "", log];
+        return [2, "", 0, log];
       }
       var dvipng_bin = init_file(prefs.getCharPref("dvipng_path"));
       if (!dvipng_bin.exists()) {
         log += "Wrong path for dvipng bin. Please set the right path in the options dialog first.\n";
-        return [2, "", log];
+        return [2, "", 0, log];
+      }
+      if (isWindows) {
```

Comment:
Also for a future PR: please add a comment here explaining why you need to go through a shell